src/python/antares_xpansion/benders_driver.py

```diff
@@ -10,20 +10,30 @@
 
 from antares_xpansion.logger import step_logger
 from antares_xpansion.study_output_cleaner import StudyOutputCleaner
+from dataclasses import dataclass
 
 
+@dataclass
+class SolversExe:
+    benders: Path
+    merge_mps: Path
+    outer_loop: Path
+
 class BendersDriver:
 
-    def __init__(self, benders, merge_mps, options_file, mpiexec=None) -> None:
+    def __init__(self, solvers_exe: SolversExe, options_file, mpiexec=None) -> None:
 
         self.oversubscribe = False
         self.allow_run_as_root = False
-        self.benders = benders
-        self.merge_mps = merge_mps
+        self.benders = solvers_exe.benders
```

Comment:
Why not keeping only a `self.solvers_exe` attribute instead of the current three ?

Reply:
SolversExe group all exe path in one stuct. It coud have been a map for exemple.
The other parameters are in a different domain
```
class paramters:
  exes: SolversExe
  option_file: Path
  mpiexe: Path
```

Reply:
Ok to keep exes packed